.github/workflows/checks.yml

```diff
@@ -41,7 +39,7 @@ jobs:
           python-version: 3.9.12
           cache: 'pip'
 
-      - name: Install legacy devnet
+      - name: Install cairo-lang
```

Comment:
this is needed for v0 compilation, right? what do you think about removing this dependency, and storing compiled contracts in repo? we will probably drop cairo0 support soon-ish, and only focus on cairo1/2